technic_worldgen/rubber.lua

```diff
@@ -25,37 +27,71 @@ minetest.register_node(":moretrees:rubber_tree_trunk", {
 	tiles = {"default_tree_top.png", "default_tree_top.png",
 		"technic_rubber_tree_full.png"},
 	groups = {tree=1, snappy=1, choppy=2, oddly_breakable_by_hand=1,
-		flammable=2},
-	sounds = default.node_sound_wood_defaults(),
+		  flammable=2, axey=1, handy=1},
```

Comment:
There are extra spaces here.
```suggestion
		flammable=2, axey=1, handy=1},
```